mldsa/poly.c

```diff
@@ -50,20 +50,21 @@ void poly_caddq(poly *a)
 
   mld_assert_bound(a->coeffs, MLDSA_N, 0, MLDSA_Q);
 }
-
-void poly_add(poly *c, const poly *a, const poly *b)
+/* Reference: `poly_add()` in the reference implementation @[REF].
```

Comment:
The function did not get renamed compared to the reference implementation. No need to mention it here.